web/src/main/java/org/cbioportal/web/error/GlobalExceptionHandler.java

```diff
@@ -1,6 +1,7 @@
 package org.cbioportal.web.error;
 
 import org.cbioportal.service.exception.CancerTypeNotFoundException;
+import org.cbioportal.service.exception.GeneNotFoundException;
```

Comment:
When we do a /genes/fetch query and supply a list of genes, I wonder what happens if one gene in a long list is not found. I'm guessing we throw the exception and there is no return value. I'm not sure if we need the use case of supplying  a list of gene identifiers and the system returns all matching genes, dropping those which fail to match. Probably not. Maybe this can be handled by an "ID" projection query? But the exception generation logic would not be projection-type sensitive so I don't think this is covered (if we even need it).